client/multi-currency/blocks/currency-switcher.js

```diff
@@ -0,0 +1,332 @@
+						) }
+					>
+						<a
+							href={
+								'/wp-admin/admin.php?page=wc-settings&tab=wcpay_multi_currency'
+							}
+							target="_blank"
+							rel="noreferrer"
+						>
+							{ __(
+								'Adjust and edit your Multi-Currency settings',
+								'woocommerce-payments'
+							) }
+						</a>
+					</PanelBody>
+					<PanelBody title={ __( 'Layout', 'woocommerce-payments' ) }>
+						<CheckboxControl
+							label={ __(
+								'Display flags',
+								'woocommerce-payments'
+							) }
+							checked={ flag }
+							onChange={ onChangeFlag }
+						/>
+
+						<CheckboxControl
+							label={ __(
+								'Display currency symbols',
+								'woocommerce-payments'
+							) }
+							checked={ symbol }
+							onChange={ onChangeSymbol }
+						/>
+
+						<CheckboxControl
+							label={ __( 'Border', 'woocommerce-payments' ) }
+							checked={ border }
+							onChange={ onChangeBorder }
+						/>
+
+						<RangeControl
+							label={ __(
+								'Border radius',
+								'woocommerce-payments'
+							) }
+							value={ borderRadius }
+							onChange={ onChangeBorderRadius }
+							min={ 1 }
+							max={ 20 }
+						/>
+					</PanelBody>
+					<PanelBody
+						title={ __( 'Typography', 'woocommerce-payments' ) }
+					>
+						<RangeControl
+							label={ __( 'Size', 'woocommerce-payments' ) }
+							value={ fontSize }
+							onChange={ onChangeFontSize }
+							min={ 6 }
+							max={ 48 }
+						/>
+						<RangeControl
+							label={ __(
+								'Line height',
+								'woocommerce-payments'
+							) }
+							value={ fontLineHeight }
+							onChange={ onChangeFontLineHeight }
+							min={ 1 }
+							max={ 3 }
+							step={ 0.1 }
+						/>
+					</PanelBody>
+					<PanelBody
+						title={ __( 'Color settings', 'woocommerce-payments' ) }
+					>
+						<ColorPaletteControl
+							onChange={ onChangeFontColor }
+							value={ fontColor }
+							label={ __( 'Text', 'woocommerce-payments' ) }
+						/>
+
+						<ColorPaletteControl
+							onChange={ onChangeBackgroundColor }
+							value={ backgroundColor }
+							label={ __( 'Background', 'woocommerce-payments' ) }
+						/>
+
+						<ColorPaletteControl
+							onChange={ onChangeBorderColor }
+							value={ borderColor }
+							label={ __( 'Border', 'woocommerce-payments' ) }
+						/>
+					</PanelBody>
+				</InspectorControls>
+
+				<div className="currency-switcher-holder" style={ styles.div }>
+					<select
+						name="currency"
+						disabled={ isLoading }
+						style={ styles.select }
+					>
+						{ isLoading && (
+							<option>
+								{ __( 'Loading…', 'woocommerce-payments' ) }
+							</option>
+						) }
+						{ ! isLoading &&
+							enabledCurrencies &&
+							enabledKeys.map( ( code ) => (
+								<option
+									key={ enabledCurrencies[ code ].id }
+									value={ enabledCurrencies[ code ].code }
+								>
+									{ flag
+										? enabledCurrencies[ code ].flag + ' '
+										: '' }
+									{ symbol
```

Comment:
If the code and symbol are equal, it displays the code twice here. Like with DKK and CHF.